packages/hoppscotch-ui/histoire.config.ts

```diff
@@ -0,0 +1,14 @@
+import { HstVue } from "@histoire/plugin-vue"
+import { defineConfig } from "histoire"
+import * as path from "path"
+import { FileSystemIconLoader } from "unplugin-icons/loaders"
+import IconResolver from "unplugin-icons/resolver"
+import Icons from "unplugin-icons/vite"
+import Components from "unplugin-vue-components/vite"
+import WindiCSS from "vite-plugin-windicss"
+
+export default defineConfig({
+  setupFile: "histoire.setup.ts",
+  plugins: [HstVue()],
+  vite(config, env) {},
```

Comment:
Underscores for unused variables ? (In this case you can just remove it though).